agent/util_windows.go

```diff
@@ -22,3 +22,9 @@ func ExecScript(script string) (*exec.Cmd, error) {
 	}
 	return cmd, nil
 }
+
+func SetSysProcAttr(cmd *exec.Cmd) {}
+
+func KillCommandSubtree(cmd *exec.Cmd) error {
```

Comment:
Can you document this behavior difference on https://www.consul.io/docs/agent/checks.html.